templates/pages/order-confirmation.html

```diff
@@ -16,7 +16,7 @@ <h1 class="is-srOnly">{{lang 'checkout.title'}}</h1>
         <h2 class="checkoutHeader-heading">
             <a class="checkoutHeader-link" href="{{urls.home}}">
                 {{#if checkout.header_image}}
-                    <img alt="{{settings.store_logo.title}}" class="checkoutHeader-logo lazyload" id="logoImage" data-sizes="auto" src="{{cdn 'img/loading.svg'}}" data-src="{{ checkout.header_image }}"/>
+                    <img alt="{{settings.store_logo.title}}" class="checkoutHeader-logo" id="logoImage" src="{{ checkout.header_image }}"/>
```

Comment:
I am fine with this change but instead of removing the lazy load we should be investigating what is causing the failure to load in the first place.

Reply:
True, I was just looking at a quick solution for now, as it's only this and another page from a previous PR. I tried researching it a bit, but it was bit over my head as to why. Thanks for reviewing, I wasn't sure as to why it wouldn't let me add reviewers.

Reply:
Do we support running theme scripts on the order confirmation page? I believe the lazysizes script just isn't being inserted there, despite being part of global.js. Assuming that's the case, this is quite possibly the correct solution, short of manually inserting a `<script>` tag in the order confirmation template.

Reply:
I am guessing that is what is happening as well.